### ISSUE TITLE
Tuplets in the space of an odd number of 32nds is not working right

### ISSUE THREAD

**Issue by DaddyLudwig** [XiaoMigros/Advanced-Tuplets#1]
So, I've been working on some Chopin, and this plugin really comes in handy for it, but it's not working right in this 1 part of Chopin's Variations on "La ci darem la mano". I just checked my version, I have the most recent one, so, it's not a version issue. It's supposed to be a 20 tuplet in the space of 11 32nds, which I calculated by going forwards and then backwards through the measure. I have been selecting the second 32nd rest, going into the plugin and setting it to 20 regular 64ths in the space of 11 regular 32nds and it gives me this weird extra 64th rest that shouldn't be there and it stops beaming properly as well. The 21 tuplet before this turned out fine, but I think that's cause it was in the space of a single longer note value. I could not split this 20 tuplet up into larger note values cause 11 is prime and there is no Add quarter duration to note notation thing. 

I tried cutting and pasting the end of measure notes somewhere else and yeah, that didn't help. Images:

Chopin Score:
![Chopin 20 tuplet](https://github.com/XiaoMigros/Advanced-Tuplets/assets/6530761/82045d34-cd60-450a-8827-9f162b6d6cc3)

Before tuplet insertion:
![MuseScore before 20 tuplet](https://github.com/XiaoMigros/Advanced-Tuplets/assets/6530761/3ec07501-a0d1-4d67-8efa-089b3221287b)

With empty tuplet:
![MuseScore Empty Tuplet](https://github.com/XiaoMigros/Advanced-Tuplets/assets/6530761/93c8283c-4b41-457a-baea-7479be8b18f3)

After note input:
![MuseScore 20 tuplet](https://github.com/XiaoMigros/Advanced-Tuplets/assets/6530761/2a977444-fee2-4587-b383-47d3c69a7bd8)

And the score so you can see what's going on:
[Variations on Là ci darem la mano Op. 2.zip](https://github.com/XiaoMigros/Advanced-Tuplets/files/12790080/Variations.on.La.ci.darem.la.mano.Op.2.zip)

**Comment by XiaoMigros** [comment]
Thanks for reporting! I've looked into the issue and can't find anything in the plugin that might be causing it, my guess is that MuseScore itself doesn't support tuplets of such length to be entered.

**Comment by DaddyLudwig** [comment]
I see, that makes sense I guess, but that now makes transcribing this Chopin piece much harder than I anticipated.

**Comment by XiaoMigros** [comment]
I'll see if I can find another way of doing it...

**Comment by DaddyLudwig** [comment]
I did it, I got the 20 tuplet in there. I first had to split the measure so that I had the 11 32nd note portion that is to be the tuplet in a separate measure from everything else. Then, I could put in a tuplet of 20 64ths in the space of 11 32nds, input the notes of the tuplet(this time it did beam correctly) and then I joined the measures together again. Guess I'll just have to keep my eye out for more of those situations where I need to split the measure to get the tuplet right.
